theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -68,7 +68,7 @@
                 {% endif %}
                 {% if user.is_authenticated and rights.statement %}
```

Comment:
I was suggesting if we can use one ```if``` statement instead of 3 nested if statements. I will leave up to you to make the final decision. Let me know I will +1.